src/components/CheckboxGroup/CheckboxGroup.stories.mdx

```diff
@@ -460,3 +465,125 @@ const onSubmit = () => {
   const CheckboxGroupValue = CheckboxGroupRef.current.value;
 };
 ```
+
+### Programmatically Update
+
+Within the `CheckboxGroup`, individual `Checkbox`es can be updated programmatically
+through the `value` prop. Like the `defaultValue` prop, the `value` prop also
```

Comment:
wording choice: replace "Like the `defaultValue`..." with "Similar to the `defaultValue`"